core/src/main/java/org/apache/spark/api/java/function/package.scala

```diff
@@ -22,4 +22,4 @@ package org.apache.spark.api.java
  * these interfaces to pass functions to various Java API methods for Spark. Please visit Spark's
  * Java programming guide for more details.
  */
-package object function 
+package object function
```

Comment:
In my PR, there is line 26.

https://github.com/apache/spark/pull/13404/files#diff-c8ebb678d9e773dd03e05b0bca473d17R26

Did I miss something? I think I'm bothering you in this PR. :)
If you don't want to update this PR. I'll reopen mine again to show it clearly.